gateway-ha/src/main/java/io/trino/gateway/ha/security/LbAuthorizer.java

```diff
@@ -2,7 +2,9 @@
 
 import io.dropwizard.auth.Authorizer;
 import io.trino.gateway.ha.config.AuthorizationConfiguration;
+import jakarta.ws.rs.container.ContainerRequestContext;
 import lombok.extern.slf4j.Slf4j;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
Is this the write annotation to use from a Trino code perspective @martint 

if not .. maybe we can switch to the right one now already instead of adding something new

Reply:
Trino is currently using `javax.annotation.Nullable`, so you could use `jakarta.anotation.Nullable` for consistency